src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.UnifiedBuild.Tasks/CreateSdkSymbolsLayout.cs

```diff
@@ -78,55 +78,31 @@ private IList<string> GenerateSymbolsLayout(Hashtable allPdbGuids)
-                    using var peReader = new PEReader(pdbStream);
-                    try
+                    string debugId = GetDebugId(guid, file);
+                    if (!allPdbGuids.ContainsKey(debugId))
                     {
-                        // Check if pdb is embedded
-                        if (peReader.ReadDebugDirectory().Any(entry => entry.Type == DebugDirectoryEntryType.EmbeddedPortablePdb))
-                        {
-                            continue;
-                        }
-
-                        var debugDirectory = peReader.ReadDebugDirectory().First(entry => entry.Type == DebugDirectoryEntryType.CodeView);
-                        var codeViewData = peReader.ReadCodeViewDebugDirectoryData(debugDirectory);
-                        guid = $"{codeViewData.Guid.ToString("N").Replace("-", string.Empty)}";
+                        filesWithoutPDBs.Add(file.Substring(SdkLayoutPath.Length + 1));
                     }
-                    catch (Exception e) when (e is BadImageFormatException || e is InvalidOperationException)
+                    else
                     {
-                        // Ignore binaries without debug info
-                        continue;
-                    }
-
-                    if (guid != string.Empty)
-                    {
-                        string debugId = GetDebugId(guid, file);
-                        if (!allPdbGuids.ContainsKey(debugId))
-                        {
-                            filesWithoutPDBs.Add(file.Substring(SdkLayoutPath.Length + 1));
-                        }
-                        else
-                        {
-                            // Copy matching pdb to symbols path, preserving sdk binary's hierarchy
-                            string sourcePath = (string)allPdbGuids[debugId]!;
-                            string destinationPath =
-                                file.Replace(SdkLayoutPath, SdkSymbolsLayoutPath)
-                                    .Replace(Path.GetFileName(file), Path.GetFileName(sourcePath));
-
-                            Directory.CreateDirectory(Path.GetDirectoryName(destinationPath)!);
-                            File.Copy(sourcePath, destinationPath, true);
-                        }
+                        // Copy matching pdb to symbols path, preserving sdk binary's hierarchy
+                        string sourcePath = (string)allPdbGuids[debugId]!;
+                        string destinationPath =
+                            file.Replace(SdkLayoutPath, SdkSymbolsLayoutPath)
+                                .Replace(Path.GetFileName(file), Path.GetFileName(sourcePath));
+
+                        Directory.CreateDirectory(Path.GetDirectoryName(destinationPath)!);
+                        File.Copy(sourcePath, destinationPath, true);
                     }
                 }
             }
 
             return filesWithoutPDBs;
         }
 
+
```

Comment:
This doesn't seem necessary/intentional.

Reply:
Unintentional - will fix.

Reply:
Fixed with https://github.com/dotnet/installer/pull/19528/commits/e431f1157fb33cb7977252e5ad9567ae53316a45